content/blog/2024/ticket-details.md

```diff
@@ -0,0 +1,90 @@
+Title: Get your PyCon India 2024 tickets now!
+Date: 2024-04-26 17:00
+Category: 2024
+Slug: announce-tickets
+Summary: The tickets for the conference are live! Grab yours before they run out.
+
+PyCon India is a volunteer-driven conference where Python communities across the nation come together to meet, discuss, and learn about their favorite aspects of Python. It's the country's premier conference for the Python programming language, unparalleled in the opportunities, knowledge, and networking potential.
+
+The conference will include a packed schedule of amazing workshops, keynotes, talks, dev sprints, and much more strung together through multiple parallel tracks. Anyone with a conference ticket is also eligible to attend the dev sprints.
+We have started selling tickets for our mega event, and details are available on the **[PyCon India 2024 website](https://in.pycon.org/2024/).**
+
+**[Get your tickets now from Townscript!](https://konfhub.com/pyconindia2024)**

```

Comment:
It's Konfhub not Townscript

Reply:
Yeah missed that. 

Also, link probably to pycon india website?

Reply:
Oops, xD.

> Also, link probably to pycon india website?

The link to the website is just above it, and below it. I'll hyperlink the image to inpycon instead of konfhub